elements/lisk-p2p/test/integration/penalty_malformed_peerinfo_list.ts

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright © 2019 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { expect } from 'chai';
+import { P2P, EVENT_BAN_PEER } from '../../src/index';
+import { createNetwork, destroyNetwork } from '../utils/network_setup';
+import { wait } from 'utils/helpers';
+
+describe('Penalty sending malformed Peer List', () => {
```

Comment:
lower case `penalty`, also change it in other places too for `describe` statements